processor/transform/compatability.ts

```diff
@@ -19,6 +20,12 @@ const compatibilityTransfomer = (): Transform => tree => {
     return SKIP;
   });
 
+  visit(tree, 'image', (node: Image, index: number, parent: Parent) => {
+    if (phrasing(parent) || !parent.children.every(child => child.type === 'image' || !phrasing(child))) return;
```

Comment:
So if I'm understanding [this thread](https://github.com/readmeio/markdown/pull/975/files#r1778953856) correctly, "phrasing content" means an inline image? (I mainly ask 'cause I was noticing some issues[^1] with inline image _editing_ earlier today and am wondering if this might help!)

[^1]: namely that the editor was auto-deleting all inline `<Images>`! 😬

Reply:
Do you have a link to a page?

Reply:
t'was a test page on the backup DB, but essentially this:

```mdx
Lorem ipsum dolor sit amet. <Image src="some.img" /> Consectetur elit neque.

Reply:
Ok, working on that in a separate PR.